roles/rhods_test_jupyterlab/tasks/main.yml

```diff
@@ -176,6 +176,39 @@
     oc delete ev -n {{ rhods_test_namespace }} --all > /dev/null
   failed_when: false
 
+- name: Conditionally update the default notebook size
+  environment:
+    KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
+  when: rhods_test_jupyterlab_notebook_cpu | length > 0
+  block:
+    - name: Retrieve the configmap containing the default notebook size
+      environment:
+        KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
+      command:
+        oc get configmap -o json --namespace redhat-ods-applications odh-jupyterhub-global-profile 
+          > {{ artifact_extra_logs_dir }}/jh_global_profile.json
+    - name: Update default notebook size in the cm file
```

Comment:
can you add an empty line between the tasks? (here and below)

Reply:
Fixed